packages/xml-views-validation/src/utils/deprecated-message-builder.ts

```diff
@@ -27,9 +28,13 @@ export function buildDeprecatedIssueMessage({
       kind = "class";
       name = ui5NodeToFQN(symbol);
       break;
+    case "UI5Aggregation":
+      kind = "aggregation";
+      name = ui5NodeToFQN(symbol);
```

Comment:
I don't think we should take the FQN of the aggregation. We only use the name everywhere else.